README.rdoc

```diff
@@ -78,6 +78,14 @@ Then add the following to the ".autotest" file:
 
     require "autotest/spec"
 
+If you use autotest/bundler, make sure your Gemfile includes
+the minitest-autotest gem. That way, the minitest-server plugin
+(which is used by minitest-autotest) will be put on the load path
+by bundler, and will be loaded by minitest. Otherwise, you may
+get the following error message:
```

Comment:
You’re right! I simplified the language in a432b3b to not distract from the real tip. Do you think the bit about the error message is okay (so someone could search and find the tip if they encounter it), or should I get rid of that too?